realdecoy/rdvue
# Add husky

### Summary of Changes

- Added husky git hook tool at version 9

### Testing Steps

- Ensure automatic linting is disabled locally by changing the editor settings in the `settings.json` file
- Make a change to the source code

![image](https://github.com/realdecoy/rdvue/assets/47512372/cc77c486-f0f5-4ca1-9d3c-1f955082a000)


- Attempt to commit the changes
- View the output

![image](https://github.com/realdecoy/rdvue/assets/47512372/c5c80691-e58c-42dd-afc8-bea756aed292)
